.buildkite/pipelines/quality-gates/pipeline.tests-staging.yaml

```diff
@@ -27,4 +27,4 @@ steps:
     command: "make -C /agent trigger-manual-verification-phase"
     if: build.branch == "main"
     agents:
-      image: "docker.elastic.co/ci-agent-images/manual-verification-agent:0.0.2"
+      image: "docker.elastic.co/ci-agent-images/manual-verification-agent:0.0.3"
```

Comment:
Updated the agent version based on the snippet provided in https://github.com/elastic/ci-agent-images/tree/main/container-images/platform-qx/manual-verification-agent

Reply:
Can we remove the `if: build.branch == "main"` conditionals for steps with the updated image?  It's built-in in 0.0.3

Reply:
Sure, removed in https://github.com/elastic/kibana/pull/167502/commits/d92613d428e859a6a82dc3e635e3f28c90c46627